evm/verify-contract.js

```diff
@@ -320,6 +320,32 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'AxelarAmplifierGateway': {
+            const AxelarAmplifierGateway = getContractJSON('AxelarAmplifierGateway');
+            const amplifierGatewayFactory = await getContractFactoryFromArtifact(AxelarAmplifierGateway, wallet);
+            const contractConfig = chain.contracts.AxelarGateway;
+            const amplifierGateway = amplifierGatewayFactory.attach(options.address || contractConfig.address);
+
+            const implementation = await amplifierGateway.implementation();
+            const previousSignersRetention = (await amplifierGateway.previousSignersRetention()).toNumber();
+            const domainSeparator = await amplifierGateway.domainSeparator();
+            const minimumRotationDelay = (await amplifierGateway.minimumRotationDelay()).toNumber();
+
+            verifyOptions.contractPath = 'contracts/gateway/AxelarAmplifierGateway.sol:AxelarAmplifierGateway';
+            await verifyContract(
+                env,
+                chain.name,
+                implementation,
+                [previousSignersRetention, domainSeparator, minimumRotationDelay],
+                verifyOptions,
+            );
+
+            verifyOptions.contractPath = 'contracts/gateway/AxelarAmplifierGatewayProxy.sol:AxelarAmplifierGatewayProxy';
```

Comment:
let's add an identifier in this contract as well to distinguish it from the base proxy

Reply:
There is already a contractId() function in this proxy. @milapsheth

Reply:
it's an internal method that's only used in the constructor of Proxy, and thus only gets included in the init code and not contract bytecode